flow/record/fieldtypes/__init__.py

```diff
@@ -640,4 +648,13 @@ class posix_path(pathlib.PurePosixPath, path):
 
 
 class windows_path(pathlib.PureWindowsPath, path):
-    pass
+    def __repr__(self) -> str:
+        s = str(self)
+        quote = "'"
+        if "'" in s:
+            if '"' in s:
+                s = s.replace("'", "\\'")
```

Comment:
I took the liberty to add some quotation tests, but i missed this one. I think this case happens if there are both single and double quotes in the path.

If you can add a test for this one it should be good to go